tests/dev/script.R

```diff
@@ -12,15 +9,28 @@ vec <- c(parameter, parameter)
 addOutputs("Organism|*|Plasma|Caffeine", simulation = sim)
 addOutputs(parameter, simulation = sim)
 
-print(outputSelections)
+#print(outputSelections)
 
 # parameter <- getParameter("Organism|Liver|Volume", sim)
 # print(parameter)
 
-# results <- runSimulation(sim, population)
+results <- runSimulation(sim)
+exportResultsToCSV(results, sim, "C:/temp/export/results.csv")
 
+pkAnalyses <- getPKAnalyses(results, sim)
 
-# getPkAnalysis(results, sim)
+exportPKAnalysesToCSV(pkAnalyses, sim, "C:/temp/export/pk.csv")
+
+pkParameters<- pkAnalyses$allPKParametersFor("Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood)")

```

Comment:
Getting all pkParameters over all indiviudals for the output Caffeine, Plasma venous blood